### ISSUE TITLE
Some bugs  in HTML code

### ISSUE THREAD

**Issue by davidwaterred** [geohot/hammer-website#7]
Hi,

I have been following you since 1 week now. I do not agree with most of your things. I am a journalist at a local newspaper.

I believe that masks are very important for covid. I have seen a lot of evidence in a lot of reputed news websites. With great power, comes a great responsibility. My friend at NYTimes is doing a story on pseudo-intilictuals who think covis is a sham, and people who are against masks, vaccines etc. She will surely mention comma.ai as well. 

Also, I think you are an anti-nationalist. You come from a hacking background, so you dont care. I hope you know hacking is illegal and unethical. It is bc of people like you Apple had to spend $$$ to improve the security. They could have used it for better things. Like paying more to foxconn workers. Hackers do not reailze that what they do is not smart - goto YT and search "how to hack gmail" you will see 100s of vids. It seemed pretty easy to me. 

Who has given you a right to hack apple or sony, costing them billiions. Who has given you the right to say covid is a sham?? You also attack Christianity in your livestreams.  You offend a lot of people .if you dont want to be a man of faith dont be. But dont make fun of others. 

PS: companies like Google are proof why America is great, you have no right to diss them.

I am not coder and my cousin who is in IT support suggested me to contact you here.

**Comment by f0ti** [comment]
Wow, we definitely need hammer. At least one to smash heads

**Comment by davidwaterred** [comment]
rise of hammer means fall of google, the whole world will laugh at our country. 
i am not against your project, but you should stop trash talking about american companies. 

also: i found someone selling a similar search engine on flippa.com for 100$. you can buy thier code and build on that. 

and why do you all use this github? why cant you use email or slack like normal people. this site is so confusing.

**Comment by terorie** [comment]
Hilarious and original, any joyers? :joy:

**Comment by roshanlam** [comment]
Not everyone is the same and @davidwaterred you should try to look at it from both sides before talking about someone. Also hacking is not illegal, unethical hacking is illegal. Unethical hacking is when someone hacks into a company and steals/leaks data or steals money from people's bank account. Ethical hacking is where people are hired by companies to hack and find vulnerabilities in their system and improve it. There are other types of hackers as well such as grey hat hackers who hack for fun. But anyway my main point is, look at both point of views before talking about anyone or anything because it can make you look stupid.

**Comment by geohot** [comment]
4chan level copypasta

**Comment by geohot** [comment]
Today when I walked into my economics class I saw something I dread every time I close my eyes. Someone had brought their new gaming laptop to class. The Forklift he used to bring it was still running idle at the back. I started sweating as I sat down and gazed over at the 700lb beast that was his laptop. He had already reinforced his desk with steel support beams and was in the process of finding an outlet for a power cable thicker than Amy Schumer's thigh. I start shaking. I keep telling myself I'm going to be alright and that there's nothing to worry about. He somehow finds a fucking outlet. Tears are running down my cheeks as I send my last texts to my family saying I love them. The teacher starts the lecture, and the student turns his laptop on. The colored lights on his RGB Backlit keyboard flare to life like a nuclear flash, and a deep humming fills my ears and shakes my very soul. The entire city power grid goes dark. The classroom begins to shake as the massive fans begin to spin. In mere seconds my world has gone from vibrant life, to a dark, earth shattering void where my body is getting torn apart by the 150mph gale force winds and the 500 decibel groan of the cooling fans. As my body finally surrenders, I weep, as my school and my city go under. I fucking hate gaming laptops.
